core/server/src/main/webapp/browse-body.jsp

```diff
@@ -16,6 +16,33 @@
 <%@ page import="static java.net.URLEncoder.encode" %>
 
 <jsp:include page="header-scripts.jsp" />
+
+<script>
+    var currentDir = "<%= request.getAttribute("currentPath").toString()%>";
+    function getInputPath() {
+        var path = $.trim($("#pathInput").val());
+        return path;
+    }
+
+    function changeDir() {
+        var path = getInputPath();
+        path = encodeURI(path);
+        window.location.href="./browse?path="+path;
+    }
+
+    $(document).ready(function () {
+        $("#pathInput").val(currentDir);
+        $("#goBtn").click(changeDir);
+        $("#pathInput").keydown(function (e) {
```

Comment:
@jsimsa , Thank you for your reply.
line 34: set input tag's value to variable currentDir's value which id is pathInput. currentDir's value is given by serverside request;
line 35: bind goBtn's click event handle function to changeDir, gotBtn is a button tag;
line 36: bind pathInput's keyDown event handle function, correctly, i want to listen enter key down event.